src/test/scala/mockws/Helpers.scala

```diff
@@ -0,0 +1,13 @@
+package mockws
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import play.api.mvc.{DefaultActionBuilder, PlayBodyParsers}
+import scala.concurrent.ExecutionContext.Implicits._
+
+object Helpers {
+  implicit val as = ActorSystem()
+  implicit val materializer = ActorMaterializer()
```

Comment:
how do we shutdown the actor system and actor materializer?
If we do not do it, then `sbt ~tests` will quickly fail with an OutOfMemory.

Which options are possible?
- using the `afterAll` specific for some test frameworks -> makes us depending on the test framework
- adding a shutdown hook. I don't know if that works with the `sbt ~test` scenario.